src/test/java/de/tum/in/www1/artemis/util/GitUtilService.java

```diff
@@ -50,48 +51,41 @@
 
     private Git localGit;
 
-    public Repository initRepo() {
+    public void initRepo() {
         try {
             deleteRepos();
-            Git remoteGit = Git.init().setDirectory(new File(remoteRoot + "/" + remoteName)).call();
-            new File(remoteRoot + "/" + remoteName + "/" + FILES.FILE1).createNewFile();
-            new File(remoteRoot + "/" + remoteName + "/" + FILES.FILE2).createNewFile();
-            new File(remoteRoot + "/" + remoteName + "/" + FILES.FILE3).createNewFile();
+            Git remoteGit = Git.init().setDirectory(new File(remoteRepositoryRoot + "/" + remoteRepositoryName)).call();
+            new File(remoteRepositoryRoot + "/" + remoteRepositoryName + "/" + FILES.FILE1).createNewFile();
+            new File(remoteRepositoryRoot + "/" + remoteRepositoryName + "/" + FILES.FILE2).createNewFile();
+            new File(remoteRepositoryRoot + "/" + remoteRepositoryName + "/" + FILES.FILE3).createNewFile();
```

Comment:
Is there a reason to explicitly concatenate the path with `"/"` here instead of using `Path.get` or `File.separator`?

Reply:
No, good catch. I think I just oberlooked it here. Will change it